modules/serverless-kn-func-emit.adoc

```diff
@@ -0,0 +1,70 @@
+[id="serverless-kn-func-emit_{context}"]
+= Emitting a test event to a deployed function
+
+You can use the `kn func emit` CLI command to emit a CloudEvent to a function that is either deployed locally or deployed to your {product-title} cluster. This command can be used to test that a function is working and able to receive events correctly.
+
+You can specify the following parameters for the emitted CloudEvent:
```

Comment:
As Zbynek says, here would be a good place to mention something like "To trigger your Function with a CloudEvent, simply run `kn func emit`.

Optionally, the CloudEvent sent to your function can be populated with custom data using the following parameters"
Or something like that.  This is the entire reason for the `kn func emit` command to exist in addition to `kn event`: the defaults.